src/components/donation/Steps/Submit/StripeCheckout/Paypal/Paypal.tsx

```diff
@@ -27,23 +22,12 @@ export default function PaypalCheckout(props: PaypalCheckoutStep) {
     splitLiq: liquidSplitPct.toString(),
   });
 
-  const dispatch = useSetter();
-
   return (
-    <Summary
-      classes={{
-        container: "flex flex-col isolate p-4 @md:p-8",
```

Comment:
nit: after this removal, all `Summary.container` props have the same classes: `grid content-start p-4 @md:p-8`; this can be hardcoded now, so there'd be no need to pass the same value in multiple places.

Reply:
Chariot (only temp disabled) also uses flex to keep chariot button at the bottom. If I hardcode this, I would extract this again once chariot is brought back.

Reply:
Leave it as-is then